README.md

```diff
@@ -1,8 +1,8 @@
 # Ramps
 
-Ramps allow lend and pay to be made with any ERC-20 token while the Diaspore contract receiving RCN their token using pooled liquidity.
+Ramps allow lend and pay to be made with any ERC-20 token while the protocol receiving RCN their token.
```

Comment:
Ramps allow lend and pay to be made with any ERC-20 token, handling conversions for the protocol to receive RCN tokens.